Custom Device Source/Docs/Modules.md

```diff
@@ -0,0 +1,87 @@
+# Modules
+
+This document lists supported modules. If you need support for a new c-Series module, or if you find an issue with the support for a specific module, please create an issue and consider committing the necessary changes and submitting a pull request. 
+
+## Analog Input
+| Model | ADC Resolution | Description | Channels | Measurement Type | Units |
+|:---:|:---:|---|:---:|:---:|:---:|
+| 9201 | 12-Bit | Analog Input | 8 | Voltage | V |
+| 9203 | 16-Bit | Analog Input | 8 | Current | A |
+| 9205 | 16-Bit | Analog Input | 32 | Voltage | V |
+| 9206 | 16-Bit | Analog Input for Fuel Cells | 32 | Voltage | V |
+| 9207 | 24-Bit | Isolated Analog Input | 8V + 8A | Voltage + Current | V , A |
+| 9208 | 24-Bit | Isolated Analog Input | 16 | Current | A |
+| 9209 | 24-Bit | 60 Vdc Isolated Analog Input | 16 | Voltage | V |
+| 9210 | 24-Bit | Thermocouple Input | 4 | Temperature or Voltage | Custom |
+| 9211 | 24-Bit | Thermocouple Input | 4 | Temperature or Voltage | Custom |
+| 9212 | 24-Bit | 250 Vrms Ch-Ch Isolation Thermocouple Input | 8 | Temperature or Voltage | Custom |
+| 9213 | 24-Bit | Thermocouple Input | 16 | Temperature or Voltage | Custom |
+| 9214 | 24-Bit | Thermocouple Input | 16 | Temperature or Voltage | Custom |
+| 9215 | 16-Bit | Simultaneous Analog Input | 4 | Voltage | V |
+| 9216 | 24-Bit | RTD Input | 8 | Temperature or Resistance | Custom |
+| 9217 | 24-Bit | RTD Input | 4 | Temperature or Resistance | Custom |
+| 9219 | 24-Bit | Universal Analog Input | 4 | Universal* | Custom |
```

Comment:
What does the asterisk indicate?

Reply:
* indicates that is not a real measurement type